yarn_api_client/base.py

```diff
@@ -16,7 +16,7 @@
 
 class Response(object):
     def __init__(self, http_response):
-        self.data = json.load(http_response)
+        self.data = json.loads(http_response.read().decode(http_response.headers.get_content_charset('utf-8')))
```

Comment:
I like how it's implemented in #7 , but there content charset is not taken into account.

Perhaps use `codecs` module in combination with `http_response.headers.get_content_charset('utf-8')` ?